javadoc: cleanup Javadoc doclint warnings

## Environment Details
* Helidon Version: 2.3.1-SNAPSHOT
* JDK version: 11

----------

## Problem Description
We have doclint errors in our javadocs (see below).  Many of these are in `messaging/connectors`.

## Steps to reproduce
1. `rm -rf target/site`
2. `mvn site`

```
[WARNING] Javadoc Warnings
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/microprofile/scheduling/src/main/java/io/helidon/microprofile/scheduling/Scheduled.java:95: warning: empty <p> tag
[WARNING] * <p>
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/microprofile/scheduling/src/main/java/io/helidon/microprofile/scheduling/Scheduled.java:189: warning: empty <p> tag
[WARNING] * <p>
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/common/configurable/src/main/java/io/helidon/common/configurable/ThreadPoolSupplier.java:299: warning: empty <p> tag
[WARNING] * <p>
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/common/configurable/src/main/java/io/helidon/common/configurable/ThreadPoolSupplier.java:344: warning: empty <p> tag
[WARNING] * <p>
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:243: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> payload(PAYLOAD payload) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:254: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, boolean value) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:265: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, byte value) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:276: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, short value) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:287: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, int value) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:298: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, long value) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:309: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, float value) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:320: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, double value) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java:331: warning: no @return
[WARNING] public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, String value) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:381: warning: no description for @param
[WARNING] * @param consumerSupplier
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:394: warning: no description for @param
[WARNING] * @param topics
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:407: warning: no description for @param
[WARNING] * @param topicPattern
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:421: warning: no description for @param
[WARNING] * @param scheduler
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:433: warning: no description for @param
[WARNING] * @param periodExecutions
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:445: warning: no description for @param
[WARNING] * @param pollTimeout
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:464: warning: no description for @param
[WARNING] * @param autoCommit
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:479: warning: no description for @param
[WARNING] * @param ackTimeout
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java:495: warning: no description for @param
[WARNING] * @param limitNoAck
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaSubscriber.java:203: warning: no description for @param
[WARNING] * @param producerSupplier
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaSubscriber.java:216: warning: no description for @param
[WARNING] * @param backpressure
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaSubscriber.java:229: warning: no description for @param
[WARNING] * @param topics
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/common/mapper/src/main/java/io/helidon/common/mapper/MapperManager.java:193: warning: no @param for priority
[WARNING] public <S, T> Builder addMapper(Mapper<S, T> mapper, Class<S> sourceType, Class<T> targetType, int priority) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/common/mapper/src/main/java/io/helidon/common/mapper/MapperManager.java:232: warning: no @param for priority
[WARNING] public <S, T> Builder addMapper(Mapper<S, T> mapper, GenericType<S> sourceType, GenericType<T> targetType, int priority) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/security/providers/idcs-mapper/src/main/java/io/helidon/security/providers/idcs/mapper/IdcsMtRoleMapperProvider.java:394: warning: empty <p> tag
[WARNING] * <p>
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/common/reactive/src/main/java/io/helidon/common/reactive/IoMulti.java:447: warning: no @return
[WARNING] public OutputStreamMultiBuilder timeout(long timeout, TimeUnit unit) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/common/reactive/src/main/java/io/helidon/common/reactive/IoMulti.java:463: warning: no @return
[WARNING] public OutputStreamMultiBuilder onRequest(BiConsumer<Long, Long> requestCallback) {
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/integrations/vault/vault/src/main/java/io/helidon/integrations/vault/spi/InjectionProvider.java:50: warning: no @param for instanceConfig
[WARNING] T apply(Vault vault, Config vaultConfig, InstanceConfig instanceConfig);
[WARNING] ^
[WARNING] /Users/jdipol/GitHub/barchetta/helidon/security/providers/common/src/main/java/io/helidon/security/providers/common/spi/AnnotationAnalyzer.java:264: warning: no @param for <T>
[WARNING] public <T> Builder register(Class<? super T> theClass, T anInstance) {
[WARNING] ^
```

--- a/common/common/src/main/java/io/helidon/common/SerializationConfig.java
+++ b/common/common/src/main/java/io/helidon/common/SerializationConfig.java
@@ -35,9 +35,10 @@ import java.util.logging.Logger;
 import java.util.stream.Collectors;
 
 /**
+ * <p>
  * Support for JEP 290 - deserialization filtering.
  * Configuration options mentioned below will differ in Helidon 3.0.0, the following table lists the options:
- * <p>
+ * </p>
  * <table class="config">
  * <caption>Configuration Options</caption>
  * <tr>
@@ -73,13 +74,14 @@ import java.util.stream.Collectors;
  * As an alternative, a file {@link #PROPERTY_FILE} can be created on the classpath with the following content, to
  * configure filter for a specific library. Do not add a global reject to these patterns!:
  * {@code pattern=oracle.sql.converter.*}
+ * </p>
  * <h3>Deserialization tracing</h3>
  * A tracing filter can be configured using system property {@value #PROP_TRACE} to log information
  * messages for each deserialization request.
  * <p>
  * To discover class patterns needed, set "no config" and "wrong config" actions to warn or ignore, and configure {@code basic}
  * tracing.
- * <p>
+ * </p>
  * Options are:
  * <ul>
  *     <li>{@code none} to disable tracing (this is the default)</li>

--- a/common/configurable/src/main/java/io/helidon/common/configurable/ThreadPoolSupplier.java
+++ b/common/configurable/src/main/java/io/helidon/common/configurable/ThreadPoolSupplier.java
@@ -313,8 +313,9 @@ public final class ThreadPoolSupplier implements Supplier<ExecutorService> {
         }
 
         /**
-         * Load all properties for this thread pool from configuration.
          * <p>
+         * Load all properties for this thread pool from configuration.
+         * </p>
          * <table class="config">
          * <caption>Optional Configuration Parameters</caption>
          * <tr>
@@ -359,7 +360,7 @@ public final class ThreadPoolSupplier implements Supplier<ExecutorService> {
          *     <td>Whether or not all core threads should be started when the pool is created.</td>
          * </tr>
          * </table>
-         * <p>
+         * <br>
          * <table class="config">
          * <caption>Experimental Configuration Parameters (<em>subject to change</em>)</caption>
          * <tr>

--- a/common/mapper/src/main/java/io/helidon/common/mapper/MapperManager.java
+++ b/common/mapper/src/main/java/io/helidon/common/mapper/MapperManager.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2021 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -186,6 +186,7 @@ public interface MapperManager {
          * @param mapper     the mapper to map source instances to target instances
          * @param sourceType class of the source instance
          * @param targetType class of the target instance
+         * @param priority   order of the mapper usage
          * @param <S>        type of source
          * @param <T>        type of target
          * @return updated builder instance
@@ -225,6 +226,7 @@ public interface MapperManager {
          * @param mapper     the mapper to map source instances to target instances
          * @param sourceType generic type of the source instance
          * @param targetType generic type of the target instance
+         * @param priority   order of the mapper usage
          * @param <S>        type of source
          * @param <T>        type of target
          * @return updated builder instance

--- a/common/reactive/src/main/java/io/helidon/common/reactive/IoMulti.java
+++ b/common/reactive/src/main/java/io/helidon/common/reactive/IoMulti.java
@@ -443,6 +443,7 @@ public interface IoMulti {
          *
          * @param timeout the maximum time to block
          * @param unit    the time unit of the timeout argument
+         * @return this builder
          */
         public OutputStreamMultiBuilder timeout(long timeout, TimeUnit unit) {
             streamMulti.timeout(TimeUnit.MILLISECONDS.convert(timeout, unit));
@@ -459,6 +460,7 @@ public interface IoMulti {
          * </ul>
          *
          * @param requestCallback to be executed
+         * @return this builder
          */
         public OutputStreamMultiBuilder onRequest(BiConsumer<Long, Long> requestCallback) {
             streamMulti.onRequest(requestCallback);

--- a/integrations/vault/vault/src/main/java/io/helidon/integrations/vault/spi/InjectionProvider.java
+++ b/integrations/vault/vault/src/main/java/io/helidon/integrations/vault/spi/InjectionProvider.java
@@ -45,6 +45,7 @@ public interface InjectionProvider {
          *
          * @param vault to use
          * @param vaultConfig configuration located on the vault node
+         * @param instanceConfig configuration of an instance
          * @return a new instance to be injected
          */
         T apply(Vault vault, Config vaultConfig, InstanceConfig instanceConfig);

--- a/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java
+++ b/messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsMessage.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Oracle and/or its affiliates.
+ * Copyright (c) 2020, 2021 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -239,6 +239,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          * Set or replace JMS payload.
          *
          * @param payload new payload
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> payload(PAYLOAD payload) {
             message.setPayload(payload);
@@ -250,6 +251,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          *
          * @param name  the name of the JMS property
          * @param value boolean value to stored as JMS property
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, boolean value) {
             properties.put(name, value);
@@ -261,6 +263,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          *
          * @param name  the name of the JMS property
          * @param value byte value to stored as JMS property
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, byte value) {
             properties.put(name, value);
@@ -272,6 +275,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          *
          * @param name  the name of the JMS property
          * @param value short value to stored as JMS property
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, short value) {
             properties.put(name, value);
@@ -283,6 +287,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          *
          * @param name  the name of the JMS property
          * @param value int value to stored as JMS property
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, int value) {
             properties.put(name, value);
@@ -294,6 +299,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          *
          * @param name  the name of the JMS property
          * @param value long value to stored as JMS property
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, long value) {
             properties.put(name, value);
@@ -305,6 +311,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          *
          * @param name  the name of the JMS property
          * @param value float value to stored as JMS property
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, float value) {
             properties.put(name, value);
@@ -316,6 +323,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          *
          * @param name  the name of the JMS property
          * @param value double value to stored as JMS property
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, double value) {
             properties.put(name, value);
@@ -327,6 +335,7 @@ public interface JmsMessage<PAYLOAD> extends Message<PAYLOAD> {
          *
          * @param name  the name of the JMS property
          * @param value string value to be stored as JMS property
+         * @return this builder
          */
         public OutgoingJmsMessageBuilder<PAYLOAD> property(String name, String value) {
             properties.put(name, value);

--- a/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java
+++ b/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaPublisher.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Oracle and/or its affiliates.
+ * Copyright (c) 2020, 2021 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -378,7 +378,7 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
          *
          * This is a mandatory parameter.
          *
-         * @param consumerSupplier
+         * @param consumerSupplier instantiated Kafka consumer
          * @return updated builder instance
          */
         public Builder<K, V> consumerSupplier(Supplier<Consumer<K, V>> consumerSupplier) {
@@ -391,7 +391,7 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
          *
          * This is a mandatory parameter if topicPattern is empty.
          *
-         * @param topics
+         * @param topics list of topics to subscribe to
          * @return updated builder instance
          */
         public Builder<K, V> topics(List<String> topics) {
@@ -400,11 +400,11 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
         }
 
         /**
-         * The list of topics to subscribe to.
+         * The pattern for selecting multiple topics.
          *
          * This is a mandatory parameter if topics is empty.
          *
-         * @param topicPattern
+         * @param topicPattern pattern for selecting multiple topics
          * @return updated builder instance
          */
         public Builder<K, V> topicPattern(Pattern topicPattern) {
@@ -418,7 +418,7 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
          *
          * This is a mandatory parameter.
          *
-         * @param scheduler
+         * @param scheduler scheduler that will read ad process messages
          * @return updated builder instance
          */
         public Builder<K, V> scheduler(ScheduledExecutorService scheduler) {
@@ -430,7 +430,7 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
          * Specifies the period in milliseconds between successive scheduler executions.
          * The default value is 100 milliseconds.
          *
-         * @param periodExecutions
+         * @param periodExecutions period in milliseconds
          * @return updated builder instance
          */
         public Builder<K, V> periodExecutions(long periodExecutions) {
@@ -442,7 +442,7 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
          * Specifies maximum time in milliseconds to block polling messages from Kafka.
          * The default value is 50 milliseconds.
          *
-         * @param pollTimeout
+         * @param pollTimeout maximum time in milliseconds to block polling
          * @return updated builder instance
          */
         public Builder<K, V> pollTimeout(long pollTimeout) {
@@ -452,16 +452,16 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
 
         /**
          * This flag defines the strategy of committing messages to Kafka.
-         * When true, the messages are committed in the moment they are polled from Kafka.
+         * When true, the messages are committed at the moment they are polled from Kafka.
          * When false, the messages are committed when {@link KafkaMessage#ack()} is invoked.
          *
-         * This value is mandatory to be specified and it must be consistent with the value enable.auto.commit
+         * This value is mandatory to be specified, and it must be consistent with the value enable.auto.commit
          * in Kafka properties. Failing to do this will result the next scenarios:
          * - For autoCommit = true and enable.auto.commit = false, messages will never be committed in Kafka.
          * - For autoCommit = false and enable.auto.commit = true, all messages will be committed and
          * {@link KafkaMessage#ack()} will have no effect.
          *
-         * @param autoCommit
+         * @param autoCommit strategy of committing
          * @return updated builder instance
          */
         public Builder<K, V> autoCommit(boolean autoCommit) {
@@ -476,7 +476,7 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
          *
          * The default value is Long.MAX_VALUE
          *
-         * @param ackTimeout
+         * @param ackTimeout time in milliseconds
          * @return updated builder instance
          */
         public Builder<K, V> ackTimeout(long ackTimeout) {
@@ -492,7 +492,7 @@ public class KafkaPublisher<K, V> implements Publisher<KafkaMessage<K, V>> {
          * The intention of this value is to fail gracefully when there are many pending commits,
          * instead of failing with OutOfMemoryError.
          *
-         * @param limitNoAck
+         * @param limitNoAck limit of messages waiting to be committed
          * @return updated builder instance
          */
         public Builder<K, V> limitNoAck(int limitNoAck) {

--- a/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaSubscriber.java
+++ b/messaging/kafka/src/main/java/io/helidon/messaging/connectors/kafka/KafkaSubscriber.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Oracle and/or its affiliates.
+ * Copyright (c) 2020, 2021 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -200,7 +200,7 @@ public class KafkaSubscriber<K, V> implements Subscriber<Message<V>> {
          *
          * This is a mandatory parameter.
          *
-         * @param producerSupplier
+         * @param producerSupplier supply instantiated the KafkaSubscriber
          * @return updated builder instance
          */
         public Builder<K, V> producerSupplier(Supplier<Producer<K, V>> producerSupplier) {
@@ -213,7 +213,7 @@ public class KafkaSubscriber<K, V> implements Subscriber<Message<V>> {
          *
          * The default value is 5.
          *
-         * @param backpressure
+         * @param backpressure number of messages requested
          * @return updated builder instance
          */
         public Builder<K, V> backpressure(long backpressure) {
@@ -226,7 +226,7 @@ public class KafkaSubscriber<K, V> implements Subscriber<Message<V>> {
          *
          * This is a mandatory parameter.
          *
-         * @param topics
+         * @param topics list of the topics
          * @return updated builder instance
          */
         public Builder<K, V> topics(List<String> topics) {

--- a/microprofile/scheduling/src/main/java/io/helidon/microprofile/scheduling/Scheduled.java
+++ b/microprofile/scheduling/src/main/java/io/helidon/microprofile/scheduling/Scheduled.java
@@ -24,12 +24,12 @@ import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
  * Marks the method to be invoked periodically according to supplied cron expression.
- * <p>
+ * <br>
  * Cron expression format:
  * <pre>{@code
  *  <seconds> <minutes> <hours> <day-of-month> <month> <day-of-week> <year>
  * }</pre>
- *
+ * <br>
  * <table>
  *  <caption><b>Cron expression fields</b></caption>
  *  <tr>
@@ -91,9 +91,7 @@ import static java.lang.annotation.RetentionPolicy.RUNTIME;
  *      </tr>
  * </tbody>
  * </table>
- *
- * <p>
- *
+ * <br>
  * <table>
  *  <caption><b>Field formats</b></caption>
  *  <tr>
@@ -159,7 +157,7 @@ import static java.lang.annotation.RetentionPolicy.RUNTIME;
  *      </tr>
  * </tbody>
  * </table>
- *
+ * <br>
  * <table>
  *  <caption><b>Examples</b></caption>
  *  <tr>
@@ -185,8 +183,6 @@ import static java.lang.annotation.RetentionPolicy.RUNTIME;
  *      </tr>
  * </tbody>
  * </table>
- *
- * <p>
  */
 @Retention(RUNTIME)
 @Target({METHOD})

--- a/security/providers/common/src/main/java/io/helidon/security/providers/common/spi/AnnotationAnalyzer.java
+++ b/security/providers/common/src/main/java/io/helidon/security/providers/common/spi/AnnotationAnalyzer.java
@@ -259,6 +259,7 @@ public interface AnnotationAnalyzer {
              *
              * @param theClass class to register the instance by
              * @param anInstance instance to register
+             * @param <T> type of the instance
              * @return updated builder instance
              */
             public <T> Builder register(Class<? super T> theClass, T anInstance) {

--- a/security/providers/idcs-mapper/src/main/java/io/helidon/security/providers/idcs/mapper/IdcsMtRoleMapperProvider.java
+++ b/security/providers/idcs-mapper/src/main/java/io/helidon/security/providers/idcs/mapper/IdcsMtRoleMapperProvider.java
@@ -394,7 +394,7 @@ public class IdcsMtRoleMapperProvider extends IdcsRoleMapperProviderBase {
 
         /**
          * Creates endpoints from provided OIDC configuration using default URIs.
-         * <p>
+         * <br>
          * <ul>
          * <li>For Asserter endpoint: {@code /admin/v1/Asserter}</li>
          * <li>For Token endpoint: {@code /oauth2/v1/token?IDCS_CLIENT_TENANT=}</li>

--- a/security/providers/idcs-mapper/src/main/java/io/helidon/security/providers/idcs/mapper/IdcsMtRoleMapperRxProvider.java
+++ b/security/providers/idcs-mapper/src/main/java/io/helidon/security/providers/idcs/mapper/IdcsMtRoleMapperRxProvider.java
@@ -440,7 +440,7 @@ public class IdcsMtRoleMapperRxProvider extends IdcsRoleMapperRxProviderBase {
 
         /**
          * Creates endpoints from provided OIDC configuration using default URIs.
-         * <p>
+         * <br>
          * <ul>
          * <li>For Asserter endpoint: {@code /admin/v1/Asserter}</li>
          * <li>For Token endpoint: {@code /oauth2/v1/token?IDCS_CLIENT_TENANT=}</li>

